src/datasources/work-orders/WorkOrdersDataSource.ts

```diff
@@ -0,0 +1,34 @@
+import { DataFrameDTO, DataQueryRequest, DataSourceInstanceSettings, TestDataSourceResponse } from '@grafana/data';
+import { BackendSrv, TemplateSrv, getBackendSrv, getTemplateSrv } from '@grafana/runtime';
+import { DataSourceBase } from 'core/DataSourceBase';
+import { WorkOrdersQuery } from './types';
+
+export class WorkOrdersDataSource extends DataSourceBase<WorkOrdersQuery> {
+  constructor (
+    readonly instanceSettings: DataSourceInstanceSettings,
+    readonly backendSrv: BackendSrv = getBackendSrv(),
+    readonly templateSrv: TemplateSrv = getTemplateSrv()
+  ) {
+    super(instanceSettings, backendSrv, templateSrv);
+  }
+
+  baseUrl = this.instanceSettings.url + '/niworkorder/v1';
+
+  defaultQuery = {};
+
+  async runQuery (query: WorkOrdersQuery, options: DataQueryRequest): Promise<DataFrameDTO> {
```

Comment:
```suggestion
  async runQuery(query: WorkOrdersQuery, options: DataQueryRequest): Promise<DataFrameDTO> {
```

Check the indentation in all places.